### ISSUE TITLE
Most elements are not labeled

### ISSUE THREAD

**Issue by heull001** [GrapheneOS/Camera#533]
Most labels, except for the tabs, are not labeled. So for visually impaired persons using TalkBack it's very very complecated to use the app.

Camera version: 83
Device: Pixel 8a
GrapheneOS build: 2025040700

**Comment by RankoR** [comment]
Fixed, will be released soon.
